src/scripts/activities/games.js

```diff
@@ -146,6 +143,21 @@ const weeklyVocab = {
 	보일러: "Heat/Heating",
 	에어컨: "Air conditioning (AC)",
 };
+
+const weeklyVocab = {
+채널: "Channel",
+뉴스: "News",
+인터넷: "Internet",
+음악: "Music",
+예술: "Art",
+보다: "To Watch",
+듣다: "To listen",
+읽다: "To read",
+티비: "TV",
+드라마: "TV Show",
+책: "Book",
+"예능 프로그램": "Variety show/program",
+};
```

Comment:
Small formatting difference here, the inside of the object should have an indent
```suggestion
const weeklyVocab = {
    채널: "Channel",
    뉴스: "News",
    인터넷: "Internet",
    음악: "Music",
    예술: "Art",
    보다: "To Watch",
    듣다: "To listen",
    읽다: "To read",
    티비: "TV",
    드라마: "TV Show",
    책: "Book",
    "예능 프로그램": "Variety show/program",
};
```

Other than that, this looks good to me! 🎉 👍

Reply:
- Reviewed -
✨ Thanks so much for doing this! ✨ 
Would you mind making those slight changes in another commit, and I'll go ahead and merge this.